frontend/src/container/MetricsApplication/constant.ts

```diff
@@ -34,6 +39,10 @@ export enum MetricsType {
 	Resource = 'resource',
 }
 
+export enum RequestMethods {
+	GETDISPATCHER = 'HTTP GET /dispatch',
+}
```

Comment:
This should not be hard coded. We should take this from the top-level operation of the selected service and use them when sending the payload. I have added two services in the collection that they change based on the service. And the value will be fetched from the topLevelOperations of service API. Let me know if this is not clear. I will be happy to take it through again.